extractors/python_functions/substring_extraction/code_snippet.md

```diff
@@ -1,11 +1,9 @@
 ```python
-from typing import List, Tuple
-
 YOUR_SUBSTRING: str = "example"
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "substring"
 
-def substring_extraction(record) -> List[Tuple[str, int, int]]:
+def substring_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.